### ISSUE TITLE
CMEMS Global reanalysis (daily)  

### ISSUE THREAD

**Issue by forbea8** [DFO-Ocean-Navigator/Ocean-Data-Map-Project#652]
I am looking to extract data from the CMEMS Global reanalysis (daily) but that model does not seem to be available. Will it become available again soon?

**Comment by dwayne-hart** [comment]
Hi,

We currently only have CMEMS Global Reanalysis (Monthly) available. We are working on getting daily data. Its unknown at this time when we will receive access to it. I will update this issue when dates are made available for end user consumption.

Best,
Dwayne

**Comment by forbea8** [comment]
Great, Thanks

I know it was working a couple of days ago as I was extracting data sets. I require a lot more data from this set for my studies.

Thanks again

Andrew
________________________________

**Comment by dwayne-hart** [comment]
Hi Andrew,

Upon further investigation. We are currently waiting on the hourly information for the same dataset. The CMEMS Global Reanalysis (Daily) information is now available for the years 1993 through to and including 2018. You may have to refresh your web browser for the Dataset menu to be updated with the new information.

Best,
Dwayne

**Comment by forbea8** [comment]
Great thanks. Any idea when the hourly data would become available? I am very interested in looking into that dataset. 

Andrew 
________________________________________

**Comment by dwayne-hart** [comment]
Waiting on data transfer and physical shipment of the drive(s) from France. It is unknown at this time when the data will arrive and made available for consumption.

**Comment by forbea8** [comment]
Hi

I have been trying to download data from the GIOPS historical data set, but for the past several days I have been encountering an internal error message. I was wondering when the data would become available again?

Regards

Andrew Forbes
________________________________

**Comment by dwayne-hart** [comment]
Hi Andrew,

You should now be able to access the THREDDS catalogs using either URL;

- http://navigator.oceansdata.ca:8080/thredds/catalog.html
- http://dory-dev.cs.dal.ca:8080/thredds/catalog.html

Best,
Dwayne

**Comment by dwayne-hart** [comment]
Closing original issue and opened a new issue regarding downloading historical datasets.
